conans/test/unittests/client/build/cmake_test.py

```diff
@@ -58,7 +58,7 @@ def config_patch_test(self):
             save(os.path.join(folder, "file3.cmake"), msg)
             save(os.path.join(folder, "sub", "file3.cmake"), msg)
 
-        cmake = CMake(conanfile, generator="Unix Makefiles")
+        cmake = CMakeBuildHelper(conanfile, generator="Unix Makefiles")
```

Comment:
This change reads a bit uncomfortable. If we are backwards compatibility, why is it necessary to modify this test? Shouldn't the previous build helper still be there and be usable? I understand maybe a change in the import for unittests, but the rest should probably remains unmodified.

Reply:
This is [unit]testing the CMake build helper class, not the `conans.CMake` class/function